iocore/net/UnixNetProcessor.cc

```diff
@@ -30,7 +30,19 @@
 // For Stat Pages
 #include "StatPages.h"
 
-int net_accept_number = 0;
+static int net_accept_number     = 0;
```

Comment:
It's not a big deal, but I think a good place for these variables is as static variables inside the functions that they're used.  The reason is that they're only used in those functions, and not meant to be touched anywhere else.